server/src/test/java/org/opensearch/index/store/remote/file/OnDemandBlockSnapshotIndexInputTests.java

```diff
@@ -301,12 +304,10 @@ public static void testSlice(OnDemandBlockSnapshotIndexInput blockedSnapshotFile
 
             newSlice.seek(0);
             assertEquals(0, newSlice.getFilePointer());
-            assertEquals(0, newSlice.currentBlockId);
```

Comment:
Note that I did remove a couple assertions around `currentBlockId` because I wanted to make that field private instead of protected. We're not losing any coverage here because computing `currentBlockPosition()` requires the `currentBlockId` so we still (indirectly) asserting on the currentBlockId value.